cli/src/main/java/keywhiz/cli/Printing.java

```diff
@@ -71,6 +63,42 @@ public void printClientWithDetails(Client client) {
     clientDetails.secrets.stream()
         .sorted(Comparator.comparing(SanitizedSecret::name))
         .forEach(s -> System.out.println(INDENT + SanitizedSecret.displayName(s)));
+
+    if (clientDetails.lastSeen == null) {
+      System.out.println("\tLast Seen: never");
+    } else {
+      String lastSeen = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss").format(
```

Comment:
I know you're not changing this here, but we inconsistently use the yyyy-mm-dd format and MM/dd/yyyy formats in keywhiz.  It would be good to standardize (on yyyy-mm-dd I think)

Reply:
For now, I'll standardize this to match the other date formats used by this class--DateFormat.getDateTimeInstance().format(<date>).